sdk/samples/iot/paho_iot_hub_pnp_component_sample.c

```diff
@@ -884,11 +897,12 @@ static az_result temp_controller_get_telemetry_message(pnp_mqtt_message* message
   az_result rc;
 
   // Get the Telemetry topic to publish the telemetry messages.
-  if (az_failed(
+  if (az_result_failed(
           rc = pnp_get_telemetry_topic(
               &hub_client, NULL, AZ_SPAN_NULL, message->topic, message->topic_length, NULL)))
   {
-    IOT_SAMPLE_LOG_ERROR("Failed to get pnp Telemetry publish topic: az_result return code 0x%08x.", rc);
+    IOT_SAMPLE_LOG_ERROR(
+        "Failed to get pnp Telemetry publish topic: az_result return code 0x%08x.", rc);
```

Comment:
I thought I applied clang-format to these, so I'm curious why this is updating.

Reply:
Line too long
![image](https://user-images.githubusercontent.com/6527137/91494583-e9370d80-e86d-11ea-8004-8183b6d1240d.png)

Reply:
I must have missed this file. ...  Perhaps I've been staring at the samples too long lol